### ISSUE TITLE
[BUG] Cannot open `archiving settings` page.

### ISSUE THREAD

**Issue by sgqy** [FreshRSS/FreshRSS#5934]
**Describe the bug**
Archiving settings not shown.

**To Reproduce**
1. Go to 'Settings - Archiving'

**Screenshots**
Blank page.

**Environment information (please complete the following information):**
- OS: `Windows 10 19044`
- Browser: `Firefox 115.5.0esr`
- FreshRSS version: `freshrss:edge-alpine -> 1.22.2-dev`
- Database version: `postgres:14-alpine`
- Installation type: `docker @ Debian 12`

**Additional context**
Original logs:
```
freshrss-freshrss-1     | [Sun Dec 10 09:15:18.399994 2023] [php:error] [pid 2229] [client 172.18.0.1:0] PHP Fatal error:  Uncaught TypeError: preg_match(): Argument #2 ($subject) must be of type string, bool given in /var/www/FreshRSS/app/Controllers/configureController.php:262\nStack trace:\n#0 /var/www/FreshRSS/app/Controllers/configureController.php(262): preg_match()\n#1 /var/www/FreshRSS/lib/Minz/Dispatcher.php(120): FreshRSS_configure_Controller->archivingAction()\n#2 /var/www/FreshRSS/lib/Minz/Dispatcher.php(51): Minz_Dispatcher->launchAction()\n#3 /var/www/FreshRSS/lib/Minz/FrontController.php(61): Minz_Dispatcher->run()\n#4 /var/www/FreshRSS/p/i/index.php(59): Minz_FrontController->run()\n#5 {main}\n  thrown in /var/www/FreshRSS/app/Controllers/configureController.php on line 262
freshrss-freshrss-1     | 172.18.0.1 - - [10/Dec/2023:09:15:18 +0000] "GET /i/?c=configure&a=archiving HTTP/1.1" 500 - "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/115.0"
```
Replace the newlines:
```
"GET /i/?c=configure&a=archiving HTTP/1.1" 500 - "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/115.0"

PHP Fatal error:  Uncaught TypeError: preg_match(): Argument #2 ($subject) must be of type string, bool given in /var/www/FreshRSS/app/Controllers/configureController.php:262
Stack trace:
#0 /var/www/FreshRSS/app/Controllers/configureController.php(262): preg_match()
#1 /var/www/FreshRSS/lib/Minz/Dispatcher.php(120): FreshRSS_configure_Controller->archivingAction()
#2 /var/www/FreshRSS/lib/Minz/Dispatcher.php(51): Minz_Dispatcher->launchAction()
#3 /var/www/FreshRSS/lib/Minz/FrontController.php(61): Minz_Dispatcher->run()
#4 /var/www/FreshRSS/p/i/index.php(59): Minz_FrontController->run()
#5 {main}
  thrown in /var/www/FreshRSS/app/Controllers/configureController.php on line 262
```

The source says:
```php
$keepPeriod = FreshRSS_Context::$user_conf->archiving['keep_period'];
if (preg_match('/^PT?(?P<count>\d+)[YMWDH]$/', $keepPeriod, $matches)) {
```
I have no knowledge to PHP, but how can I find `$user_conf`?
I did not find similar keywords like `keep_period` in DB nor Browser storage...

**Comment by Alkarex** [comment]
Regression from https://github.com/FreshRSS/FreshRSS/pull/5830
Fix coming

**Comment by Alkarex** [comment]
Fixed with https://github.com/FreshRSS/FreshRSS/pull/5935
Could you please try again?

**Comment by sgqy** [comment]
> Fixed with #5935 Could you please try again?

Awesome, it works. 🎉
